auth/src/main/java/org/openedx/auth/presentation/signin/SignInViewModel.kt

```diff
@@ -177,4 +203,34 @@ class SignInViewModel(
             }
         } ?: onUnknownError()
     }
+
+    fun openLink(fragmentManager: FragmentManager, links: Map<String, String>, link: String) {
```

Comment:
```
    fun openLink(fragmentManager: FragmentManager, title: String, link: String) {
        router.navigateToWebContent(fragmentManager, title, link)
    }

Reply:
we shouldn't paste logic in the UI level. So that's why all my logic is executed in the view model